lib/vector-tap/src/lib.rs

```diff
@@ -171,57 +183,194 @@ impl<'a> TapRunner<'a> {
             match message {
                 Ok(Some(Some(res))) => {
                     if let Some(d) = res.data {
-                        for tap_event in d.output_events_by_component_id_patterns.iter() {
-                            match tap_event {
-                                GraphQLTapOutputEvent::Log(ev) => {
-                                    println!(
-                                        "{}",
-                                        self.formatter.format(
-                                            ev.component_id.as_ref(),
-                                            ev.component_kind.as_ref(),
-                                            ev.component_type.as_ref(),
-                                            ev.string.as_ref()
-                                        )
-                                    );
-                                }
-                                GraphQLTapOutputEvent::Metric(ev) => {
-                                    println!(
-                                        "{}",
-                                        self.formatter.format(
-                                            ev.component_id.as_ref(),
-                                            ev.component_kind.as_ref(),
-                                            ev.component_type.as_ref(),
-                                            ev.string.as_ref()
-                                        )
-                                    );
-                                }
-                                GraphQLTapOutputEvent::Trace(ev) => {
-                                    println!(
-                                        "{}",
-                                        self.formatter.format(
-                                            ev.component_id.as_ref(),
-                                            ev.component_kind.as_ref(),
-                                            ev.component_type.as_ref(),
-                                            ev.string.as_ref()
-                                        )
-                                    );
-                                }
-                                GraphQLTapOutputEvent::EventNotification(ev) => {
-                                    if !quiet {
-                                        eprintln!("{}", ev.message);
-                                    }
+                        let output_events: Vec<GraphQLTapOutputEvent> = d
+                            .output_events_by_component_id_patterns
+                            .into_iter()
+                            .filter(|event| {
+                                !matches!(
```

Comment:
Question for my own understanding: 

This filter removes events of type `GraphQLTapOutputEvent::EventNotification` when `quiet` is `true`, right? I wonder how useful `quiet` is for the `OutputChannel::AsyncChannel` use case, but I guess it's OK to keep as is.

Reply:
Yea unsure how common this `EventNotification` type is if it requires `quiet` flag, but will keep the behavior consistent with `stdout` for now